setup.cfg

```diff
@@ -0,0 +1,32 @@
+[metadata]
+name = ops.manifest
+version = 1.0.0
+author = Adam Dyess
+author_email = adam.dyess@canonical.com
+description = "Kubernetes manifests for Operators"
+long_description = file: README.md, CHANGELOG.md
+long_description_content_type = text/markdown
+keywords = juju, charming, kubernetes, operators, manifests, yaml
+license = MIT License
```

Comment:
should there be a license.txt at the root of the repo?

Reply:
oo... lets do that